docs/development/building/Building-in-Ubuntu.md

```
@@ -62,6 +62,55 @@ This will start a local server for the Betaflight App.
+# Android development (Betaflight Configurator)
+export JAVA_HOME=/opt/android-studio/jbr
+export ANDROID_HOME="$HOME/Android/Sdk"
+export NDK_HOME="$ANDROID_HOME/ndk/$(ls -1 $ANDROID_HOME/ndk 2>/dev/null | head -1)"
+export PATH="$PATH:$ANDROID_HOME/platform-tools:$NDK_HOME"
+' >> ~/.bashrc
+source ~/.bashrc
+```
+
+You can now build and run the Android app:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)